specification/inference/_types/CommonTypes.ts

```diff
@@ -1497,7 +1497,11 @@ export class GoogleVertexAIServiceSettings {
 
 export enum GoogleModelGardenProvider {
   google,
-  anthropic
+  anthropic,
```

Comment:
Could you also add in the `url` and `streaming_url` field comments which providers require which ones? (basically the same thing we did in the comment in the Java code).